.pre-commit-config.yaml

```
@@ -5,7 +5,7 @@ repos:
       hooks:
           - id: black
             language_version: python3
-    - repo: https://gitlab.com/pycqa/flake8
+    - repo: https://github.com/pycqa/flake8
       rev: 3.8.3
```

```suggestion
      rev: 5.0.4
```
This seems to be the latest, so I suggest changing that as well.